frontend/src/components/projectSection/projectCard.css

```diff
@@ -4,7 +4,11 @@
     text-shadow: var(--theme-text-shadow);
 }
 
-.project-separator {}
+.project-separator {
+    font-family: var(--theme-font);
```

Comment:
Isn't this only used without any test?

Reply:
ah, oops I forgot to make the separator not so hacky. 1 second